.changelog/2842.feature.md

```diff
@@ -0,0 +1,12 @@
+Add new consensus-related Prometheus metrics
+
+Four new metrics have been added:
+
+- `oasis_worker_epoch_number` is the current epoch number as seen by the
+  worker.
+- `oasis_registry_node_registered` is a binary metric which denotes, if the
+  node is registered validator.
```

Comment:
i think the code just checks if the node is registered (not a registered validator)